### PR TITLE
Rearranged facility home layout

### LINKED ISSUE #7285: Improving UI of Facility dashboard
The current facility dashboard (https://care.coronasafe.in/facility/7ab27974-32c8-4e36-bdf9-dbf102524718) is 
Screenshot 2024-02-27 at 2 43 59 PM

Make the following changes to improve the usability of the screen

- [ ] rearrange the banner on top with basic details of the hospital as below:
a) arrange the address and phone number along with the rest of the details
b) Enlarge the image. retain aspect ratio of 1:1
c) reduce the spacing between "manage facility" and "central nursing station" and reduce height of the entire banner.
Screenshot 2024-02-27 at 2 52 28 PM

- [ ] Below the banner with basic details of the hospital, there are cards for "Oxygen information" "bed capacity", "Doctors list" and "corona Triage". rearrange these as 
1. "Bed capacity"
2. "Doctors List"
3. "Oxygen information"
4. "corona Triage"

### PR DESCRIPTION
**Pull request by kshitijv256**
## Proposed Changes

- Fixes #7285 
- Changed banner layout
- Rearranged other sections

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

#### Mobile 

Screenshot from 2024-03-20 22-12-38

#### Tablet

Screenshot from 2024-03-20 22-15-50

#### Laptop 14 inch

Screenshot from 2024-03-20 22-11-43

#### 24 inch 2K screen

Screenshot from 2024-03-20 22-12-02

### CODE REVIEW

**Review by Ashesh3 — APPROVED** [review]

**Review by rithviknishad — APPROVED** [review]

**Comment by nihal467** [comment]
image

@aparnacoronasafe feel like there are a lot of white spacing issues occurring, can you look into the UI once again

**Comment by aparnacoronasafe** [comment]
> image
> 
> @aparnacoronasafe feel like there are a lot of white spacing issues occurring, can you look into the UI once again

@nihal467 whats your screen size? I am on a 13inch mac.

This is what I am seeing at 100% screen size


I think it is good to go

**Comment by nihal467** [comment]
@aparnacoronasafe I am using 15.6-inch laptop screen as well as 27-inch monitor screen , on both of them, they are having a huge white spacing issue

**Comment by aparnacoronasafe** [comment]
@nihal467 lets release this and think of iterative improvements in design?

**Comment by rithviknishad** [comment]
IMO the current staging itself seems to be a better layout than the one in this PR for both resolutions.

**Comment by nihal467** [comment]
@gigincg can you review this

**Comment by gigincg** [comment]
@kshitijv256 Can you update the PR based on the Image Reference in the Issue. Padding for the Picture, and the Alignment of the Buttons. Also, post Screenshots for Mobile, Tab, HD & Full HD Screens. Let's wrap this ASAP.

**Comment by kshitijv256** [comment]
@gigincg @nihal467 @rithviknishad updated screenshots for different screen sizes

**Comment by rithviknishad** [comment]
There needs to be a a justify-between between the manage facility and rest of the buttons in desktop view as per the issue ref. image

image

**Comment by kshitijv256** [comment]
@rithviknishad  `justify-between` makes the space too large
should I add some fixed gap between instead, or push this one with `justify-between` ?

Screenshot from 2024-03-21 16-53-14

**Comment by rithviknishad** [comment]
Please refer the ref. screenshot in the issue

image

The vertical height of the div containing the buttons is limited to the same height as the facility picture.

**Comment by kshitijv256** [comment]
Changed as requested @rithviknishad 

Screenshot from 2024-03-22 19-00-36

**Comment by rithviknishad** [comment]
Layout and alignments looks good to me, but the Edit image button on hover no longer seems to be present

Develop deployment on hover:
image

This branch on hover:
image

**Comment by rithviknishad** [comment]
Can we also give the image a `rounded-lg` border?

**Comment by kshitijv256** [comment]
@rithviknishad tooltip shows when you hover on top part of image

Screenshot from 2024-03-23 12-46-50

File: src/Components/Facility/FacilityHome.tsx [hunk 1]
```diff
@@ -106,7 +106,7 @@ export const FacilityHome = (props: any) => {
   const editCoverImageTooltip = hasPermissionToEditCoverImage && (
     <div
       id="facility-coverimage"
-      className="absolute right-0 top-0 z-10 flex h-full w-full flex-col items-center justify-center bg-black text-sm text-gray-300 opacity-0 transition-opacity hover:opacity-60 md:h-[88px]"
+      className="absolute right-0 top-0 z-10 flex h-full w-full flex-col items-center justify-center rounded-t-lg bg-black text-sm text-gray-300 opacity-0 transition-[opacity] hover:opacity-60 md:h-[88px]"
```
Comment (khavinshankar) [inline src/Components/Facility/FacilityHome.tsx]: @kshitijv256 please double check `transistion-[opacity]` class

image

Comment (kshitijv256) [reply src/Components/Facility/FacilityHome.tsx]: @khavinshankar 
I missed it as I was trying out other transitions there, changed it back to `transition-opacity`

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**